src/wavm/WAVMWasmModule.cpp

```diff
@@ -250,26 +234,13 @@ bool WAVMWasmModule::tearDown()
 
     executionContext = nullptr;
 
-    if (compartment == nullptr) {
-        return true;
-    }
-
-    bool compartmentCleared =
-      Runtime::tryCollectCompartment(std::move(compartment));
-    if (!compartmentCleared) {
-        logger->warn("Failed GC for compartment");
-    } else {
-        logger->debug("Successful GC for compartment");
+    if (compartment != nullptr) {
+        // Release build complains that this is unused as the assertion is
+        // removed
+        [[maybe_unused]] bool compartmentCleared =
+          Runtime::tryCollectCompartment(std::move(compartment));
+        assert(compartmentCleared);
```

Comment:
This `[[maybe_unused]]` is to fix an error when doing the release build, where it complains that `compartmentCleared` is unused (because the `assert` is no-oped)